bynder_sdk/client/asset_bank_client.py

```diff
@@ -5,6 +5,7 @@ class AssetBankClient:
     """ Client used for all the operations that can be done to the
     Bynder Asset Bank.
     """
+
```

Comment:
I'll comment here since it's a bit easier to have a "threaded" conversation this way:

Something that would be great to address with this PR is the fragility of the upload process regarding failed requests. With both the old and the new code uploads are basically ruined when a single request fails which is not very user friendly. If say for example chunk 50000/50001 fails, we should retry it rather than going "welp your upload failed, try again". Same principle goes for the finalise endpoint (and perhaps media save endpoint if a retry would make sense there).

Reply:
As discussed we will pick this up in another ticket.

Reply:
Alright, great observation I must say. This makes good sense to do to me.